jsettlers.logic/src/jsettlers/logic/movable/Movable.java

```diff
@@ -60,9 +60,30 @@
 	private static final ConcurrentLinkedQueue<Movable> allMovables = new ConcurrentLinkedQueue<Movable>();
```

Comment:
Please do not do that ;-)

Only reorder or re-format if there is a real significance of doing it. This will only lead to merge conflicts with other branches, especially the donkey branch.